src/System.ValueTuple/tests/System.ValueTuple.Tests.builds

```diff
@@ -2,10 +2,16 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="System.ValueTuple.Tests.csproj" />
```

Comment:
Nevermind I see you have it below. For consistency please keep the default builds first.